module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -508,28 +515,23 @@ namespace module::input {
                         sensors->feet[BodySide::LEFT].down  = true;
 
                         std::array<bool, 2> feet_down = {true};
```

Comment:
This initialisation is misleading and should be amended. I presume that it's meant to be `true, true`?
```suggestion
                        std::array<bool, 2> feet_down = {true, true};
```